tools/nxstyle.c

```diff
@@ -967,8 +968,8 @@ int main(int argc, char **argv, char **envp)
 
                   rhcomment = -1;
 
-                  if (ncomment > 0 && (!strncmp(&line[ii], "if", 2)
-                      || !strncmp(&line[ii], "el", 2)))
+                  if (ncomment > 0 && (strncmp(&line[ii], "if", 2) == 0
+                      || strncmp(&line[ii], "el", 2) == 0))
```

Comment:
In all other places in nxstyle.c, the conditional is broken with the || at the end of the preceding line, not at the beginning of the following line, I don't think this is addressed by the coding standard, but it would be nice to be consistent within the documents.